### PR TITLE
Docs review

### PR DESCRIPTION
**Pull request by lauriejim**
#### Description of what you did:

Here is a global review of the docs. But as you know many thing should still be improved.
Part of my obj and after this review, I know items we have to improved.
Waiting also for our tech writer to help me on that point.

### CODE REVIEW

File: docs/v3.x/getting-started/installation.md [hunk 1]
```diff
@@ -1,6 +1,12 @@
-# ⚙️ Installation
+# Installation
 
-### Installation guides
+Strapi gives you many possible installation options for your project or application. Strapi can be installed on your computer or services such as Digital Ocean or Platform.sh. The following documentation covers differents options to install strapi and getting sttarted.
+
+::: tip
+For a more detailed overview of deployment please see the [related documentation](./deployment).
```
Comment (derrickmehaffy) [inline docs/v3.x/getting-started/installation.md:6]: Not sure if this needs a .html or not, since we have so many issues with other links.

Comment (lauriejim) [reply docs/v3.x/getting-started/installation.md:6]: Need to add `.md` you are right

File: docs/v3.x/getting-started/installation.md [hunk 2]
```diff
@@ -1,6 +1,12 @@
-# ⚙️ Installation
+# Installation
 
-### Installation guides
+Strapi gives you many possible installation options for your project or application. Strapi can be installed on your computer or services such as Digital Ocean or Platform.sh. The following documentation covers differents options to install strapi and getting sttarted.
```
Comment (derrickmehaffy) [inline docs/v3.x/getting-started/installation.md:3]: > Strapi gives you many possible installation options for your project or application. Strapi can be installed on your computer or services such as DigitalOcean, Amazon AWS, or Platform.sh. The following documentation covers many different options to install Strapi and getting started on using it.

File: docs/v3.x/plugins/upload.md [hunk 3]
```diff
@@ -346,7 +357,7 @@ When configuring your upload provider you might want to change the configuration
 
 You can set a specific configuration in the `./config/env/{env}/plugins.js` configuration file and it will be used to overwrite the one in the default configuration.
 
-## Create providers
+## Create a providers
```
Comment (derrickmehaffy) [inline docs/v3.x/plugins/upload.md:360]: Remove the s in providers

File: docs/v3.x/plugins/upload.md [hunk 4]
```diff
@@ -300,16 +300,27 @@ You can check all the available providers developed by the community on npmjs.or
...
+:::
+
+::: tab npm
 
 ```
-$ yarn add strapi-provider-upload-aws-s3
+npm install strapi-provider-upload-aws-s3 --save
 ```
 
+<<<<<<< HEAD
```
Comment (alexandrebodin) [inline docs/v3.x/plugins/upload.md:319]: git conflict

**Pull request merged**